lib/controllers/playlists/favoritesController.js

```diff
@@ -7,27 +7,45 @@ const index = (request, response) => {
 }
 
 const create = async (req, res) => {
-  const playlist = await database('playlists').where({id: req.params.playlist_id}).first()
-  console.log(playlist)
-  const favorite = await database('favorites').where({id: req.params.favorite_id}).first()
-  database('playlist_favorites')
-    .insert({
-      playlistId: playlist.id,
-      favoriteId: favorite.id
-    })
-    .then(() =>{
-      res.status(201).json({Success: `${favorite.title} has been added to ${playlist.title}`})
-    })
-    .catch(error => {
-      console.log(error)
-      res.status(500).json({error: error})
-    })
+  const playlistId = req.params.playlist_id
+  const favoriteId = req.params.favorite_id
+
+  if (isNaN(playlistId) || isNaN(favoriteId)) { 
+    console.log('HERERASDFASDF')
```

Comment:
What does this do?

Reply:
I was having difficulties reaching that point. The console.log just needs removed. I think you can sense my frustration with it...